resources/events.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Events Calendar
+order: 1
+---
+
+# Events Calendar
+<iframe id="open-web-calendar" 
+    style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+    src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fics.teamup.com%2Ffeed%2Fks3qwzicncjfkkeyq6%2F8703308.ics"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation"
+    allowTransparency="true" scrolling="no" 
+    frameborder="0" height="700px" width="120%"></iframe>
```

Comment:
Why 120%? Does this work on mobile?

Reply:
Good question. Checking now.

Reply:
Works but looks veeeryy weird
<img width="364" alt="Screen Shot 2020-09-23 at 18 05 12" src="https://user-images.githubusercontent.com/11277600/94031241-55525600-fdc7-11ea-9ac0-77c04f22686e.png">